src/watch/index.ts

```diff
@@ -35,6 +35,10 @@ const flags = {
 		type: [String],
 		description: 'Paths & globs to exclude from being watched',
 	},
+	add: {
```

Comment:
Rename to `include`.

Can you also add an `exclude` flag that's identical to `ignore`?

Reply:
Okay, but does the use of exlude conflict with the original ignore?

Reply:
Modified, see if it meets expectations, and I also added the noIgnore flag, otherwise some requirements may not be met, such as hidden file `.env`